### PR TITLE
Add deletion of validatingwebhookconfigurations to stuck uninstall

### PR DESCRIPTION
**Pull request by oernii**

### CODE REVIEW

File: content/docs/installation/kubectl.md [hunk 1]
```diff
@@ -117,6 +117,11 @@ First, delete existing cert-manager webhook configurations, if any:
 kubectl delete mutatingwebhookconfigurations cert-manager-webhook
 ```
 
+Then try do delete the corresponding validatingwebhookconfigurations
+```bash
+kubectl delete validatingwebhookconfigurations cert-manager-webhook
```
Comment (irbekrm) [inline content/docs/installation/kubectl.md:122]: Could we just move this line to the already existing code block under the line that deletes mutating configurations?

**Review by irbekrm — CHANGES_REQUESTED** [review]
Thanks for the PR @oernii !

I've added a small suggestion, otherwise happy to approve. 

We could also add the same section to helm uninstall doc too https://github.com/cert-manager/website/blob/master/content/docs/installation/helm.md#namespace-stuck-in-terminating-state

File: content/docs/installation/kubectl.md (hunk 1 above)
Comment (oernii) [reply content/docs/installation/kubectl.md:122]: Sure

Comment (inteon) [reply content/docs/installation/kubectl.md:118]: done

**Pull request merged**